### ISSUE TITLE
Launchpad: Glossary (Part 2) - Automate Launchpad's glossary

### ISSUE THREAD

**Issue by odadacharles** [canonical/open-documentation-academy#351]
The beginning of this work must be preceded by the successful completion of https://github.com/canonical/open-documentation-academy/issues/350 .

The Launchpad Team would like to add a glossary feature to the [Launchpad Manual](https://ubuntu.com/docs/launchpad/), similar to [Ubuntu Server's](https://ubuntu.com/server/docs/reference/glossary/). Once a list of seed words has been collected, defined, and added to a new glossary doc, a feature should be added to enable users to read definitions simply by hovering over specific words that appear in the glossary. This is already implemented in the Ubuntu Server docs linked above. 

Feel free to ask more questions regarding this task. 

GitHub Repo: https://github.com/canonical/launchpad-manual
